pow/src/main/java/tech/pegasys/teku/pow/Web3jEth1Provider.java

```diff
@@ -245,8 +244,13 @@ public SafeFuture<Boolean> validate() {
 
   private SafeFuture<Result> validateChainId() {
     return getChainId()
-        .thenApply(
-            chainId -> {
+        .handle(
+            (chainId, ex) -> {
+              if (ex != null) {
+                LOG.info(
+                    "eth_chainId method is not supported by provider, skipping validation", ex);
```

Comment:
I'd skip this log - it will just create a lot of noise. Maybe set it to trace level.